src/hyperfine/benchmark.rs

```diff
@@ -201,6 +201,22 @@ fn run_cleanup_command(
     run_intermediate_command(shell, command, show_output, error_output)
 }
 
+#[cfg(unix)]
+fn extract_exit_code(status: ExitStatus) -> Option<i32> {
+    use std::os::unix::process::ExitStatusExt;
+
+    /* From the ExitStatus::code documentation:
+       "On Unix, this will return None if the process was terminated by a signal."
+       In that case, ExitStatusExt::signal should never return None.
+    */
+    status.code().or_else(|| status.signal())
```

Comment:
I think this should either return the exit code (if it's there) OR the signal n + 128. This is a standard procedure, e.g. in Bash: https://tldp.org/LDP/abs/html/exitcodes.html

Returning just the signal number (e.g. `9` for `KILL`) could be confused with the actual exit code 9.

Reply:
You are right! I could not easily find a constant in `libc` or in the `std` library, so I hardcoded `128` with a comment.